tt_metal/llrt/llrt.cpp

```diff
@@ -322,6 +322,12 @@ void wait_until_cores_done(
             }
         }
         loop_count++;
+        // Continuously polling cores here can cause other host-driven noc transactions (dprint, watcher) to drastically
```

Comment:
when do we see this?  I think we only hit this code path w/ slow dispatch and firmware init.  SD is a don't care.  firmware init...on galaxy, I wonder if this delay is measurable, probably not.  Is there another time this shows up?

Reply:
Issue arises in slow dispatch dprint tests (seen here: https://github.com/tenstorrent/tt-metal/issues/9615). Uses of `wait_until_cores_done()`:

* Clean reset init
* FW init
* Device close (wait for dispatch kernels to finish)
* Slow Dispatch via LaunchProgram/WaitProgramDone

Only the last one is where we ran into issues (dprint/watcher server are off I think at the time the other calls are made).

Reply:
Just leave it